File: 1_class_object.cpp
#include<bits/stdc++.h>
using namespace std;

class A{
public:
    // constructor
    A(){
        printf("I am Constructor\n");
    }

    // method
    void display(){
        cout<<"I am Display Method\n";
    }

    // destructor
    ~A(){
        cout<<"I am Destructor\n";
    }
};


int main(){
    A obj;
    obj.display();

    return 0;
}

File: Week_0/1_class_object.py
class A:
    # constructor
    def __init__(self) -> None:
        print("I am Constructor")
    
    # method
    def display(self):
        print("I am Display Method")

    # destructor
    def __del__(self):
        print("I am Destructor")

    def __str__(self) -> str:
        return "I am Class A"


obj = A()
obj.display()


